SETUP/manual_web/split_test/horizontal_split_flex.php

```diff
@@ -0,0 +1,59 @@
+<?php
+$relPath="../../../pinc/";
+include_once($relPath.'base.inc'); // require_login()
+include_once($relPath.'slim_header.inc');
+
+// Demonstrate two pane vertical split
+
+require_login();
```

Comment:
I would remove this from all files and allow unauthenticated people to use it.

Reply:
done this.